isn&#39;t it related to <a href="https://svn.open-mpi.org/trac/ompi/ticket/1469">https://svn.open-mpi.org/trac/ompi/ticket/1469</a> ?<br><br><div><span class="gmail_quote">On 6/30/08, <b class="gmail_sendername">Lenny Verkhovsky</b> &lt;<a href="mailto:lenny.verkhovsky@gmail.com">lenny.verkhovsky@gmail.com</a>&gt; wrote:</span><blockquote class="gmail_quote" style="margin:0;margin-left:0.8ex;border-left:1px #ccc solid;padding-left:1ex">
<div>I am <span id="st" name="st" class="st">not</span> familiar with spawn test of IBM, but maybe this is&nbsp;right behavior,</div>
<div>if spawn test allocates 3 ranks on the node, and then allocates another 3 </div>
<div>then this test suppose to fail due to max_slots=4.</div>
<div>&nbsp;</div>
<div>But it fails with the fallowing hostfile as well BUT WITH A DIFFERENT ERROR.</div>
<div>&nbsp;</div>
<div>#cat hostfile2&nbsp;<span class="q"><br>witch2 slots=4 max_slots=4<br></span>witch3 slots=4 max_slots=4</div>
<div>witch1:/home/BENCHMARKS/IBM # /home/USERS/lenny/OMPI_ORTE_18772/bin/mpirun -np 3 -hostfile hostfile2 dynamic/spawn<br>bash: <span id="st" name="st" class="st">orted</span>: <span id="st" name="st" class="st">command</span> <span id="st" name="st" class="st">not</span> <span id="st" name="st" class="st">found</span><br>
[witch1:22789] --------------------------------------------------------------------------<br>
A daemon (pid 22791) died unexpectedly with status 127 while attempting<span class="q"><br>to launch so we are aborting.</span></div><span class="q">
<div>There may be more information reported by the environment (see above).</div>
</span><div><span class="q">This may be because the daemon was unable to find all the needed shared<br>libraries on the remote node. You may set your LD_LIBRARY_PATH to have the<br>location of the shared libraries on the remote nodes and this will<br>

automatically be forwarded to the remote nodes.<br>--------------------------------------------------------------------------<br></span>[witch1:22789] --------------------------------------------------------------------------<br>

mpirun was unable to cleanly terminate the daemons on the nodes shown<br>below. Additional manual cleanup may be required - please refer to<br>the &quot;orte-clean&quot; tool for assistance.<br>--------------------------------------------------------------------------<br>

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; witch3 - daemon did <span id="st" name="st" class="st">not</span> report back when launched<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <br></div><div><span class="e" id="q_11ad934bf685b398_9">
<div class="gmail_quote">On Mon, Jun 30, 2008 at 9:38 AM, Lenny Verkhovsky &lt;<a href="mailto:lenny.verkhovsky@gmail.com" target="_blank" onclick="return top.js.OpenExtLink(window,event,this)">lenny.verkhovsky@gmail.com</a>&gt; wrote:<br>

<blockquote class="gmail_quote" style="padding-left:1ex;margin:0px 0px 0px 0.8ex;border-left:#ccc 1px solid">
<div>Hi, </div>
<div>trying to run mtt I failed to run IBM spawn test. It fails only when using hostfile, and not when using host list.</div>
<div>( OMPI from TRUNK )</div>
<div>&nbsp;</div>
<div>This is working :</div>
<div>#mpirun -np 3 -H witch2 dynamic/spawn</div>
<div>&nbsp;</div>
<div>This Fails:</div>
<div># cat hostfile<br>witch2 slots=4 max_slots=4</div>
<p>#mpirun -np 3 -hostfile hostfile dynamic/spawn<br>[witch1:12392] --------------------------------------------------------------------------<br>There are not enough slots available in the system to satisfy the 3 slots<br>

that were requested by the application:<br>&nbsp; dynamic/spawn</p>
<p>Either request fewer slots for your application, or make more slots available<br>for use.<br>--------------------------------------------------------------------------<br>[witch1:12392] --------------------------------------------------------------------------<br>

A daemon (pid unknown) died unexpectedly on signal 1&nbsp; while attempting to<br>launch so we are aborting.</p>
<p>There may be more information reported by the environment (see above).</p>
<div>This may be because the daemon was unable to find all the needed shared<br>libraries on the remote node. You may set your LD_LIBRARY_PATH to have the<br>location of the shared libraries on the remote nodes and this will<br>

automatically be forwarded to the remote nodes.<br>--------------------------------------------------------------------------<br>mpirun: clean termination accomplished</div>
<div>&nbsp;</div>
<div>&nbsp;</div>
<div>Using hostfile1 also works</div>
<div>#cat hostfile1</div>
<div>witch2</div>
<div>witch2</div>
<div>witch2</div>
<div>&nbsp;</div>
<div>&nbsp;</div>
<div>Best Regards</div>
<div>Lenny.</div></blockquote></div><br>
</span></div></blockquote></div><br>

